src/harness/infra/README.md

```diff
@@ -18,7 +18,7 @@ Shared Infrastructure components of Harness
 | postgresql.auth.existingSecret | string | `"postgres"` |  |
 | postgresql.commonLabels.app | string | `"postgres"` |  |
 | postgresql.fullnameOverride | string | `"postgres"` |  |
-| postgresql.image.tag | string | `"14.7.0-debian-11-r28"` |  |
+| postgresql.image.tag | string | `"14.8.0-debian-11-r17"` |  |
```

Comment:
@shaswatdeep  does this version change affect any of our external HA story ? do we keep versions same ?